dev/setup-rancher-clusters

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-set -euxo pipefail
```

Comment:
out of curiosity, why dropping pipefail?

Reply:
Not sure but the pipe fails.
```
++ curl https://releases.rancher.com/server-charts/latest/index.yaml
++ yq -r '.entries | to_entries | .[].value[] | .version' latest.yaml
++ sort -V
++ tail -1
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 15813    0 15813    0     0  49249      0 --:--:-- --:--:-- --:--:-- 49415
curl: (56) Failure writing output to destination, passed 5484 returned 0
+ version=2.9.1-rc6
```

I don't care too much since this is a dev script 🤷